### ISSUE TITLE
property_enum_coverage false positive when a #define is used for property name

### ISSUE THREAD

**Issue by Tamaranch** [bilelmoussaoui/gobject-lint#128]
Here is an example: https://gitlab.xfce.org/xfce/xfce4-power-manager/-/blob/bb6789aae21f1543ac359b1761b037a9faba3dad/src/xfpm-power.c#L1041

Warning:
```
src/xfpm-power.c:128:1: warning: Property enum value 'PROP_PRESENTATION_MODE' is declared but never installed in xfpm_power_class_init [property_enum_coverage]
```

Replacing `PRESENTATION_MODE` with `"some-string"` in the first parameter of `g_param_spec_boolean` silences the warning.

Version: git main e098446de41005ae189b9ea1fe1010be62f9ab22

**Comment by bilelmoussaoui** [comment]
I don't see where that is defined anywhere in the codebase https://gitlab.xfce.org/search?search=PRESENTATION_MODE&nav_source=navbar&project_id=14&group_id=2&search_code=true&repository_ref=bb6789aae21f1543ac359b1761b037a9faba3dad. 

The tool assumes you would pass the property name directly and that has been the case for almost every codebase i tried it on... Maybe you can fix the code instead?

**Comment by Tamaranch** [comment]
It's defined here: https://gitlab.xfce.org/xfce/xfce4-power-manager/-/blob/429df9fec8f9331812d79bec8c74d6fb86f4bbb5/common/xfpm-config.h#L71

No, I'm not going to fix the code :) Using a `#define` instead of a string directly is perfectly valid anywhere. But I can understand that gobject-linter doesn't want to support this specific case: I've added an entry to my false positives file, and that's good enough for me.

NB: if other properties are defined in the same way below this one, gobject-linter "doesn’t see them": it doesn’t report any other warnings, only for the first one.

**Comment by bilelmoussaoui** [comment]
Should be fixed now

**Comment by Tamaranch** [comment]
Yes I can confirm it's fixed, thank you!
